.drone.yml

```diff
@@ -2982,6 +2982,50 @@ steps:
       - mkdir -p /go/artifacts
       - aws s3 sync s3://$AWS_S3_BUCKET/teleport/tag/${DRONE_TAG##v}/ /go/artifacts/
 
+  - name: Download RPM repo contents
+    image: amazon/aws-cli
+    environment:
+      AWS_S3_BUCKET:
```

Comment:
Will this result in environment variables set in the container? Can we instead mount it to `~/.aws/credentials` of the container @webvictim

Reply:
It will result in those environmental variables being set in the container for this particular pipeline step, yes. We could write them to `~/.aws/credentials`, but then they end up stored on disk which isn't much better. What's the thinking here?